proxy/coco/tests/common.rs

```diff
@@ -0,0 +1,97 @@
+use std::{path::PathBuf, time::Duration};
```

Comment:
`fixtures` might be a good name for this module to be inline with `librad`. Just a suggestion though :)

Reply:
It's not only fixtures tho, also a bunch of utility.

Reply:
:+1: